CharacterCreation/03_Feats.txt

```diff
@@ -1152,6 +1152,27 @@ to the weapon's largest range bracket. Miss Chance can go below the -2 minimum.
 
 Steady Shot may be taken multiple times, up to a maximum of 4; the bonuses stack.
 
+== Staying Strikes ==
+
+Prerequisites:
+
+*  Feat: Unarmed Combat Expertise
+
+Cooldown: 2 Turns
+Duration: Until the end of your next turn
+
+    Your talent and training in melee combat allow you to maneuver your footwork
+and time your strikes to prevent an enemy from advancing along their chosen path.
+The extra speed and power you get from your nanites makes this technique even
+more effective.
+
+    When you hit an enemy with a melee attack, either during your turn or as an
+opportunity attack, you may choose to trigger this ability. If you do so, the
+target must make a Shock Check. If they succeed, they take 20 extra damage of 
+the same type that your weapon deals. If they fail, their speed is reduced to 0
+until the end of your next turn. 

```

Comment:
This is a little odd in that there are situations where you would activate the feat and hope they beat the save because you want the extra damage. Keeping an enemy in melee range as a melee class is enough of an upside that I'm not sure we need anything to happen in the fail case.